README.md

```diff
@@ -1,4 +1,17 @@
-# iModel.js Samples
+# 🚨Repository deprecated🚨
+**This repository has been deprecated and is no longer maintained. Please refer to three separate "starter apps" for examples of iModel.js applications.**
+### Web viewer
+[itwin-viewer](https://github.com/imodeljs/itwin-viewer)

```

Comment:
Do we want to point to the Getting Started tutorial instead? The root of the iTwin Viewer repo is rather difficult to figure out what to do first and tutorial may be a better spot.

Reply:
Swapped all to link to tutorials